flags/flags.go

```diff
@@ -10,16 +10,93 @@
 //
 // SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
 
-//go:build !things
-// +build !things
-
 package flags
 
 import (
 	"flag"
 
+	"github.com/eclipse-kanto/suite-connector/flags"
+	"github.com/eclipse-kanto/suite-connector/logger"
+
 	"github.com/eclipse-kanto/azure-connector/config"
 )
 
-// AddHub adds the Hub connection related flags.
-func AddHub(_ *flag.FlagSet, _, _ *config.AzureSettings) {}
+const (
+	flagCACert           = "cacert"
+	flagTenantID         = "tenantId"
+	flagIDScope          = "idScope"
+	flagSASTokenValidity = "sasTokenValidity"
+)
+
+// AddGlobal adds the Cloud Agent global flags.
+func AddGlobal(f *flag.FlagSet) (configFile *string) {
+	return flags.AddGlobal(f)
+}
+
+// ConfigCheck checks for config file existence.
+func ConfigCheck(logger logger.Logger, configFile string) {
+	flags.ConfigCheck(logger, configFile)
+}
+
+// Add adds the Cloud Agent flags and uses the provided settings to collect the provided values.
```

Comment:
- Cloud Agent -> azure connector
- `provided` is duplicated, the wording can be polished a little